docs/federate.md

```diff
@@ -98,6 +98,74 @@ _matrix._tcp.<server_name>``. In our example, we would expect this:
 Note that the target of a SRV record cannot be an alias (CNAME record): it has to point
 directly to the server hosting the synapse instance.
 
+### Delegation FAQ
+#### When do I need a SRV record or .well-known URI?
+
+If your homeserver listens on the default federation port (8448), and your
+`server_name` points to the host that your homeserver runs on, you do not need an SRV record or `.well-known/matrix/server` URI.
+
+For instance, if you registered `example.com` and pointed its DNS A record at a
+fresh Upcloud VPS or similar, you could install Synapse on that host,
```

Comment:
```suggestion
fresh server, you could install Synapse on that host,
```

(or "machine", or whatever the correct word in English is)

Not sure it's worth getting too specific here, or including other brands in our documentation.